app/javascript/controllers/time_zone_controller.js

```diff
@@ -0,0 +1,48 @@
+import { Controller } from "@hotwired/stimulus"
+import lookupFriendlyTimeZone from "util/lookupFriendlyTimeZone"
+
+export default class extends Controller {
+  connect() {
+    const resolvedZone = Intl.DateTimeFormat().resolvedOptions().timeZone
```

Comment:
is `resolvedZone` via [resolvedOptions().timeZone](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/resolvedOptions) a way to get a user's default time zone? not fully sure how `Intl.DateTimeFormat()` works, but since nothing is passed in here, i'm assuming it either defaults to UTC or the user's time zone?

Reply:
The docs state it's either a String, or undefined if it cannot be determined.  (Though I've only seen timezone strings testing locally - but this handles undefined as well).